src/why3prop.ml

```diff
@@ -0,0 +1,61 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Why3
+
+(* a type to present the list of why3 constants and lambdapi terms *)
+type cnst_table = (term * Term.lsymbol) list
+
+(** [t_goal g] output the translation of a goal [g] into a why3.*)
+let rec t_goal : string -> term -> bool = fun sp g ->
+    match Basics.get_args g with
+    | (Symb({sym_name="P"; _}, _), [t])     ->
```

Comment:
You should use builtins instead of hard-coded strings here. See how it is done in rewrite.ml.